### DISCUSSION TITLE
How to return List with custom methods

### DISCUSSION THREAD

**Discussion by thewh1teagle** [PyO3/pyo3#3492] [Questions]
Hello everyone
I'm trying to return List of dict where the list have custom methods to manipulate the data

in python it looks like:
```python
class CustomList(list):
  def __str__(self): ...
  def to_some_other_object():
    for i in self...
```

You can see what I tried here
Basically I want to be able to use method `to_cookiejar` on the Objects I return from the `pyfunctions` there
[lib.rs#L10](https://github.com/thewh1teagle/rookie/blob/main/bindings/python/src/lib.rs#L10)

**Comment by davidhewitt** [comment]
Sorry for the delay in response. At the moment extending `list` is not quite supported due to the same reasons `tuple` doesn't work, see #1647. In principle this can work, just needs some time investment.
